dev-tools/mage/crossbuild.go

```diff
@@ -185,6 +205,9 @@ func crossBuildImage(platform string) (string, error) {
 		tagSuffix = "darwin"
 	case strings.HasPrefix(platform, "linux/arm"):
 		tagSuffix = "arm"
+		if runtime.GOARCH == "arm64" {
+			tagSuffix = "base-arm-debian9"
+		}
```

Comment:
Is it possible to publish docker images for different architectures with the same tag (with multi-architecture manifests), if we do this we don't need to add logic here to chose the image depending on the local architecture, docker does it.

We can go on as this by now, but I'd suggest to publish multi-architecture manifests for maintainability.